src/main/resources/entity-types/users/drv_simple_user_details.json5

```diff
@@ -422,11 +466,19 @@
       },
       queryable: true,
       visibleByDefault: false,
-      valueGetter: "(\n  SELECT\n    array_agg(pf.value::text) FILTER (\n      WHERE\n        (pf.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(src_users_users.jsonb->'proxyFor'::text) pf (value)\n)",
-      filterValueGetter: "(\n  SELECT\n    array_agg(lower(pf.value::text)) FILTER (\n      WHERE\n        (pf.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(src_users_users.jsonb->'proxyFor'::text) pf (value)\n)",
+      sourceAlias: "user",
+      valueGetter: "(\n  SELECT\n    array_agg(pf.value::text) FILTER (\n      WHERE\n        (pf.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(:sourceAlias.jsonb->'proxyFor'::text) pf (value)\n)",
+      filterValueGetter: "(\n  SELECT\n    array_agg(lower(pf.value::text)) FILTER (\n      WHERE\n        (pf.value::text) IS NOT NULL\n    ) AS ARRAY_AGG\n  FROM\n    jsonb_array_elements_text(:sourceAlias.jsonb->'proxyFor'::text) pf (value)\n)",
       valueFunction: 'lower(:value)',
     }
   ],
+  "sources": [
+    {
+      "type": "db",
+      "target": "src_users_users",
+      "alias": "user"
+    }
+  ],
```

Comment:
Everything looks good, the only minor nit is that we can leave the double quotes off the keys (e.g. "type" could be type, "sources" could be sources). And the values could use single quotes if they don't contain single quotes within the string. So (sourceAlias: "user" could be sourceAlias: 'user'). Not a big deal but it would keep things consistent with the other fields.